packages/bbui/src/Form/Core/Multiselect.svelte

```diff
@@ -81,7 +81,10 @@
   const makeToggleOption = (map: Record<string, boolean>, value: string[]) => {
     return (optionValue: string) => {
       if (map[optionValue]) {
-        const filtered = value.filter(option => option !== optionValue)
+        // comparison needs to take into account different types, always compare them as strings
+        const filtered = value.filter(
+          option => option.toString() !== optionValue.toString()
```

Comment:
Going by the types, these values should already be a string and not need the `toString` ?

```ts
value: string[]
```

```ts
return (optionValue: string)
```

Is it possible to prevent the wrong types being passed into the toggle function in the first place? Or update the type signatures to reflect the real possible types?